index.html

```diff
@@ -43,7 +43,141 @@ <h4 class="connectmywork"> LET’S CONNECT</h4>
+            </ul>
+    <p class="section1"> A daily selection of privately personalized reads; no accounts or sign-ups required </p>
+</div>
+        <div >
+            <ul class="program1">
+                <li class="Program-skilhtml">html</li>
+                <li class="Program-skilhtml">css</li>
+                <li class="skiljava">javascript</li>
+            </ul>
+        </div>
+        <div class="buttonsee">
+            <button class="button1"> See Project</button>
+        </div>
+</div>
+  </section>
+  <section class="Workssection">
+    <div class="cardBack">
+    <a class="card1" href="#"><img src="./image/Snapshoot Portfolio1 (2).png" alt=""></a>
+        <h3  class="tonic2" >Multi-Post Stories</h3>
+    <div>
+            <ul class="works">
+                <li class="canopy"> CANOPY</li>
+                <li class="BackendDev">BackendDev</li>
+                <li class="num"> 2015 </li>
+            </ul>
+            <p class="section1"> A daily selection of privately personalized reads; no accounts or sign-ups required </p>
+    </div>
+    <div >
+        <ul class="program1">
+            <li class="Program-skilhtml">html</li>
+            <li class="Program-skilhtml">css</li>
+            <li class="skiljava">javascript</li>
+        </ul>
+   </div>
+<div class="buttonsee">
+    <button class="button1"> See Project</button>
+</div>
+   
+</div>
+  </section>
+  <section class="Workssection">
+    <div class="cardBack">
+    <a class="card3" href="#"></a>
+        <h3  class="tonic" >Tonic</h3>
+    <div>
+            <ul class="works">
+                <li class="canopy"> CANOPY</li>
+                <li class="BackendDev">BackendDev</li>
+                <li class="num"> 2015 </li>
+            </ul>
+    <p class="section1"> A daily selection of privately personalized reads; no accounts or sign-ups required </p>
+    </div>
+    <div >
+        <ul class="program1">
+            <li class="Program-skilhtml">html</li>
+            <li class="Program-skilhtml">css</li>
+            <li class="skiljava">javascript</li>
+        </ul>
+    </div>
+        
+    <div class="buttonsee">
+        <button class="button1"> See Project</button>
+    </div>
+     </div>
+  </section>
+  <section class="Workssection">
+    <div class="cardBack">
+    <a class="card4" href="#"></a>
     
+        <h3  class="tonic3" >Multi-Post Stories </h3>
+        <div>
+    <ul class="works">
+        <li class="canopy"> CANOPY</li>
+        <li class="BackendDev">BackendDev</li>
+        <li class="num"> 2015 </li>
+     </ul>
+    <p class="section1"> A daily selection of privately personalized reads; no accounts or sign-ups required </p>
+</div>
+<div >
+    <ul class="program1">
+        <li class="Program-skilhtml">html</li>
+        <li class="Program-skilhtml">css</li>
+        <li class="skiljava">javascript</li>
+    </ul>
+</div>
+   <div class="buttonsee">
+    <button class="button1"> See Project</button>
```

Comment:
- [ ] I would kindly suggest you to  stick to the design as much as possible (e.g., font, colors, images, text, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1).As yu can see your `see project button` not the same as the figma design. you can reduce the padding/margin also make it align with the modal text  and "LETS CONNECT TEXT" to fit the original design so tha your website will look professional and user friendly.see the screenshot below
- 
![Screenshot 2023-06-10 230915cc](https://github.com/tes-nat/My_Portofolio/assets/92458236/f337c8bf-25e6-41a5-a36d-db47fcf33d07)


- [ ] figma design
![image](https://github.com/tes-nat/My_Portofolio/assets/92458236/f761d3dc-d957-45bd-bf5e-da013327cd88)